Azure/sonic-buildimage
# [device/celestica]: Add xcvrd event support for Seastone-DX010

**- Why I did it**
- The xcvr service requires an event detection function, unplug or plug in the transceiver.

Fix #5853

**- How I did it**
- Add sysfs interrupt to notify userspace app of external interrupt
- Implement get_change_event() in chassis api.

**- How to verify it**
- Verify xcvrd running with error in syslog
- Verify port status and port information with this following command
  - `show interfaces status`
  - `show interfaces transceiver eeprom`
- Log : [xcvrd.log](https://github.com/Azure/sonic-buildimage/files/5534763/xcvrd.log)


**Signed-off-by**: Wirut Getbamrung [wgetbumr@celestica.com]
